### PR TITLE
issue-207

### PR DESCRIPTION
**Pull request by mishaone**
* Updated working with pipelines section in the UG
* Reused a topic from GS doc in UG
* Random idea to remove conditionals by renaming images so did that for all modules I could think of

### CODE REVIEW

**Comment by mishaone** [comment]
@Preeticp also was this what you meant about naming images? If so, sorry I didn't understand it at the time. What I've done here is added {context} to the images so that the hello world version of the image will always be named hello-world_new_dialog.png while the spring boot version will be spring-boot_new_dialog.png so that means we have one line of code instead of a full conditional.

**Comment by mishaone** [comment]
The build here failed at the same weird point as the other one, but the docs build step was fine so I'll go ahead and merge this.

File: docs/topics/modules/configuring_pipelines.adoc [hunk 1]
```diff
@@ -5,14 +5,19 @@ Advanced users can change the settings for their project pipeline builds in Open
 
 .Prerequisites
 
-. Ensure that you have created a build pipeline for your project. See <<creating_a_new_pipeline>> for instructions.
+. Ensure that you have created a build pipeline for your project. See <<creating_new_project-user-guide>> for instructions.
```
Comment (rkratky) [inline docs/topics/modules/configuring_pipelines.adoc]: This introduced a mistake. Will fix in #227.

File: docs/topics/modules/using_stack_report.adoc [hunk 2]
```diff
@@ -26,7 +26,7 @@ At the Deployment stage, {osio} Analytics analyzes your stack and provides a det
 
 .Prerequisites
 
-Ensure that you deploy your project and <<creating_a_new_pipeline, create a build pipeline>> for your project.
+Ensure that you deploy your project and <<creating_new_project-user-guide>> for your project.
```
Comment (rkratky) [inline docs/topics/modules/using_stack_report.adoc]: This introduced a mistake. Will fix in #227.

Comment (rkratky) [reply docs/topics/modules/using_stack_report.adoc]: I'm struggling to understand this change. The sentence would read:

_Ensure that you deploy your project and **Creating a new project** for your project._

Why was `creating_a_new_pipeline.adoc` deleted?

Comment (rkratky) [reply docs/topics/modules/using_stack_report.adoc]: OK, I get it now. This was not easy to decipher...

**Pull request merged**